src/js/index.js

```diff
@@ -96,14 +122,18 @@ window.addEventListener('load', () => {
 
 	if ('URLSearchParams' in window) {
 		search = new URLSearchParams(window.location.search);
+
 		if (search.has('tags')) {
 			selectr.setValue(search.get('tags').split(' '));
 		}
+
+		showExpired.checked = search.has('expired');
```

Comment:
If we go `expired=y`, this should be changed to `(search.get('expired') || null) === 'y'` or something.

Reply:
instead of 'y' or 'n', can we use '1' / '0'?

Reply:
> instead of 'y' or 'n', can we use '1' / '0'?

In my opinion, that would make the URL unreadable and ambiguous. I think `y` and `n` are simple, succinct and easily understandable, and so we should keep using those.

Reply:
> If we go `expired=y`, this should be changed to `(search.get('expired') || null) === 'y'` or something.

Why do we need the `|| null` part?

Reply:
It's not needed, get returns null if it doesn't find a matching value.